.gitignore

```diff
@@ -7,3 +7,4 @@
 build
 dist
 htmlcov
+.idea
```

Comment:
**suggestion (non-blocking):** Maybe better to have this file in your global `.gitignore`.
```shell
$ git config --global core.excludesfile ~/.gitignore
$ echo .idea >> ~/.gitignore
```